### ISSUE TITLE
Notification light stops working while charging

### ISSUE THREAD

**Issue by atithi02** [fakemanoan/LineageOS-Releases#89]
### Build used

lineage-19.1-20240515-UNOFFICIAL-zerofltexx.zip 

### Model number

SM-G920F

### Describe the bug

While the phone is charging, the notification lights stop working and from that moment the LED does not indicate incoming messages, missed calls or charging.
[radio-bat.txt](https://github.com/fakemanoan/LineageOS-Releases/files/15460483/radio-bat.txt)
[logcat-bat.txt](https://github.com/fakemanoan/LineageOS-Releases/files/15460509/logcat-bat.txt)

### To Reproduce

1-Turn on the 3G network.
2-Discharge your phone completely.
3-Connect to charge and turn on the phone.
4-Turn off data transmission.
5-The LED will turn off during or when the charge is complete.

### Any extra info

This doesn't happen every time you charge. It also happens with data transmission enabled.
The same thing happened in the February, March and April 2024 releases.
After restarting the phone, the notifications start working.

**Comment by Duxss** [comment]
Yeah it happen often, i thought i was only one since no one mentioned it till now :}
